iterable-extension/src/main/java/com/mparticle/ext/iterable/IterableExtension.java

```diff
@@ -28,6 +28,7 @@
     public static final String SETTING_USER_ID_FIELD = "userIdField";
     public static final String USER_ID_FIELD_CUSTOMER_ID = "customerId";
     public static final String USER_ID_FIELD_MPID = "mpid";
+    public static final String PLACEHOLDER_EMAIL_DOMAIN = "@placeholder.email";
```

Comment:
Since you're extracting this into a constant, could you also update `getPlaceholderEmail` to reference it?